src/main/java/Duke.java

```diff
@@ -1,10 +1,220 @@
+                            if (Arrays.asList(keys_entry).size() == 2) {
+                                successFlag = true;
+                                ent = new list_Entry_Deadline(keys_entry[0], false, keys_entry[1]);
+                            }
+                        } else if (keys[0].equalsIgnoreCase("event")) {
+                            currentHint = commandHints.EVENT;
+
+                            String[] keys_entry1 = keys[1].split(" /from ", 2);
+                            if (Arrays.asList(keys_entry1).size() == 2) {
+                                String[] keys_entry2 = keys_entry1[1].split(" /to ", 2);
+                                if (Arrays.asList(keys_entry2).size() == 2) {
+                                    successFlag = true;
+                                    ent = new list_Entry_Event(keys_entry1[0], false, keys_entry2[0], keys_entry2[1]);
+                                }
+                            }
+                        }
+                    }
+
+                    if (successFlag) {
+                        user_list.add(ent);
+                        StringBuilder text = new StringBuilder();
+                        String count_msg = "okie! Added a task to the list:\n   " + ent + "\nNow you have " + user_list.size() + " tasks in the list.";
+                        text.append(count_msg);
+                        bot_functions.shifted_print(text.toString());
+                    } else {
+                        bot_functions.shifted_print("Wrong format");
+                    }
+
+                }
+            }
+        }
+        System.exit(0);
+    }
+
+
+    public static class bot_functions {
+        public static final String HORIZONTAL_LINE = "____________________________________________________________\n";
+
+        public static String greetingString(String name){
+            String msg = "";
+            msg += String.format("%30s", "（„• ֊ •„)♡\n");
+            msg += "heyo! my name is ✦" + name + "✦\n";
+            msg += "What would you like for me to do?\n";
+            return msg;
+        }
+
+        public static String sad_cat_img() {
+            return
+                    "⠀               />    フ\n" +
+                            "               | 　_  _|\n" +
+                            "           ___/` ミ＿_xノ\n" +
+                            "          /　　　     |\n" +
+                            "         /　 ヽ　     ﾉ\n" +
+                            "        │　    |　|　|\n" +
+                            "     /￣|　    |　|　|\n" +
+                            "    | (￣ ヽ＿ヽ_)__)\n" +
+                            "    ＼二二)⠀⠀\n";
+        }
+
+
+
+
+        public static String signoffString(){
+            String msg = "";
+            msg += "okay then, bye\n";
+            msg += sad_cat_img();
+
+            return msg;
+        }
+
+        public static void shifted_print(String text) throws Exception{
```

Comment:
May have to change naming to fit function name standards in Java? 

```suggestion
public static void shiftPrint(String text) throws Exception { 
```

Reply:
you can also consider adding javaDoc here since shiftPrint is not a very clear indication of what this function is doing